cluster-autoscaler/core/scaledown/latencytracker/node_latency_tracker.go

```diff
@@ -94,7 +94,10 @@ func (t *NodeLatencyTracker) Process(autoscalingCtx *ca_context.AutoscalingConte
 		nodeName := unremovableNode.Node.Name
 		if info, exists := t.unneededNodes[nodeName]; exists {
 			duration := time.Since(info.unneededSince)
-			metrics.UpdateScaleDownNodeRemovalLatency(false, duration)
+			latency := duration - info.removalThreshold
```

Comment:
This is almost identical to the other block of code below. Perhaps an opportunity to remove some duplication?